src/ad-sizes.ts

```diff
@@ -0,0 +1,87 @@
+export type GuAdSizeString = 'fluid' | `${number},${number}`;
```

Comment:
Where should types like these live? I'd prefer to keep them as close as possible to the functions that use them rather than define them globally but could see why we might want to define them in `./types.ts`

Reply:
Close is good!